### PR TITLE
migrated to null safety

### PR DESCRIPTION
**Pull request by venkata-reddy-dev**
### Hello @Benjith , I am contributing following changes

**1) migrated to dart sound null safety**
**2) converted example flutter project to  standalone dart project** 

**Note : package version bumped to 0.1.0**

Please review this pull request changes

### CODE REVIEW

File: example/.gitignore [hunk 1]
```diff
@@ -1,71 +1,9 @@
...
-*.ipr
-*.iws
-.idea/
-
-# Visual Studio Code related
-.vscode/
-
-# Flutter/Dart/Pub related
-**/doc/api/
+# Files and directories created by pub
```
Comment (kaladron) [inline example/.gitignore:1]: I'd just update to the latest .gitignore template from flutter.   A lot of the defaults are useful (like the .vscode, etc)

File: pubspec.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 name: number_to_words
 description: Convert number into words by localeID
-version: 0.0.3
+version: 0.1.0
```
Comment (kaladron) [inline pubspec.yaml]: This would be an API breaking change full null safety.  Using Semvar, it should roll the major verison.

**Review by venkata-reddy-dev — COMMENTED** [review]
looks good

**Review by venkata-reddy-dev — COMMENTED** [review]
agreed

**Comment by venkata-reddy-dev** [comment]
@Benjith  dart null safety support is blocking in flutter 2.0 projects. colud you please look into this pull request

**Comment by venkata-reddy-dev** [comment]
@kaladron  major version change updated and raised a new  [pull](https://github.com/Benjith/NumberToWords/pull/4) request

**Pull request closed without merging**